install/installer/cluster_dev.py

```diff
@@ -256,23 +388,26 @@ def parse_cli_args():
     if cli.git_user_name:
         UserNameValidator.validate(cli.git_user_name, interactive=False)
 
+    if cli.cloud and cli.cloud_provider:
+        if cli.cloud_provider not in CLOUD_PROVIDERS[cli.cloud]:
+            _exit_(
+                f'Cloud provider can be: {CLOUD_PROVIDERS[cli.cloud_provider]}, ' +
+                f'but provided: {cli.cloud_provider}',
+            )
+
     return cli
 
 
-def get_git_username(cli_arg, git):
+@typechecked
+def get_git_username(git: object) -> str:
```

Comment:
Nobody uses `object`. You can often see `Any`, though. But that's if you're not sure about the type. Do you know what it is?
```suggestion
def get_git_username(git: typing.Any) -> str:
```